cfgov/unprocessed/css/molecules/global-eyebrow.less

```diff
@@ -0,0 +1,105 @@
+/* topdoc
+  name: Social Media
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-social-media">
+            <h5 class="m-social-media_heading">Share this</h5>
+            <ul class="list">
+                <li class="list_item">
+                    <a class="m-social-media_icon" href="#">
+                        <span class="cf-icon cf-icon-email-social-square"></span>
+                        <span class="u-visually-hidden">
+                            Share by email
+                        </span>
+                    </a>
+                </li>
+            </ul>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-social-media
+            .m-social-media_heading
+            ul.list
+              li.list_item
+                .m-social-media_icon
+                  span.cf-icon
+                  span.u-visually-hidden
+  tags:
+    - cfgov-molecules
+*/
+
+.m-global-eyebrow {
+    .webfont-regular();
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/ed2b42ad78a1b8c384073a0a321387ec2be2587a